portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,10 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds comments from /data
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function getComments() {
+    const commentsData = await fetch("/data");
+    const comments = await commentsData.text();
```

Comment:
I don't think you need this second await. Thoughts?

Reply:
I defer to your expertise here. I followed an example from the tutorial that looked like this

```javascript
/**
 * Another way to use fetch is by using the async and await keywords. This
 * allows you to use the return values directly instead of going through
 * Promises.
 */
async function getRandomQuoteUsingAsyncAwait() {
  const response = await fetch('/random-quote');
  const quote = await response.text();
  document.getElementById('quote-container').innerText = quote;
}
```

I just prefer the async/await syntax over promises 😅

Reply:
What I meant is that I think the second await seems unnecessary since the first one will pause the function to wait for the response. So what would there be left to wait for in line 20?

Reply:
This is the explanation they gave https://github.com/Anthony-Fiddes/portfolio/blob/style-stretch/walkthroughs/week-3-server/step-2-fetch-walkthrough.md#handleresponse